dependency-inversion-android/README.md

```diff
@@ -0,0 +1,9 @@
+# Dependency Inversion
+
+Throughout this exercise YOU CANNOT CHANGE THE IMPLEMENTATIONS OF InternetFruitRepository or Preferences! However, you can make them implement interfaces.
+
+Go to the `CachingFruitFetcherTests`. First try to write the three tests without touching the current code. Hard, right?
+
+Now use dependency inversion on the _dependencies_ of the `fetchFruit()` method (there are two). Now can you write the tests?
```

Comment:
some links to some info on dependency inversion would be good.

Also you could use our DI image Qi made ;)